site/src/pages/release-notes/1.26.3.mdx

```diff
@@ -0,0 +1,20 @@
+---
+date: 2023-11-17
+---
+
+## 🛠️ Bug fixes
+
+- You no longer see a `NullPointerException` when an <type://Endpoint> is selected from an <type://EndpointGroup> #5302
+  - This bug affects 1.26.2 only.
+- You no longer see an intermittent deadlock when a <type://StreamMessage> is being aborted. #5301
+- Updated the GraalVM `native-image` configuration. #5296 #5308
+
+## 🙇 Thank you
+
+<ThankYou usernames={[
+  'ikhoon',
+  'jrhee17',
+  'minwoox',
+  'trustin',
+  'yswdqz'
```

Comment:
```suggestion
  'trustin'
```

Reply:
I believe this is the reporter of the issue https://github.com/line/armeria/issues/5296 which I think is worth crediting